src/components/sound-editor/sound-editor.css

```diff
@@ -149,6 +149,15 @@ $border-radius: 0.25rem;
     margin-bottom: -0.375rem;
 }
 
+/* mirror the louder/softer speaker icons when rtl */
```

Comment:
Should the waves on the echo flip?
![image](https://user-images.githubusercontent.com/2855464/44860560-7aa22380-ac44-11e8-8ff2-433193fabdb8.png)